### PR TITLE
Update the Contributor guide, architecture diagram and presentation videos

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description

This PR makes the following changes - 
1. It updates the architecture diagram to also include the latest components introduced to Vitess (VTOrc and Vtadmin)
2. Update the presentation videos page to include links to the latest presentations delivered at conferences (KubeCon China 2021, KubeCon EU 2022, Percona Live 2022, KubeCon NA 2022)
3. Update the contributor guide to represent the recent changes made to the requirements for PRs and reviewers guide.

## Related Issues
- https://github.com/vitessio/website/issues/872

### CODE REVIEW

File: content/en/docs/resources/presentations.md [hunk 1]
```diff
@@ -4,6 +4,34 @@ weight: 1
 aliases: ['/resources/presentations.html', '/docs/resources/presentations/']
 ---
 
+## KubeCon NA 2022
+
+One VTOrc To Rule Them All – High Availability In a Distributed Database System - Deepthi Sigireddi & Manan Gupta, PlanetScale, Inc.
```
Comment (deepthi) [inline content/en/docs/resources/presentations.md]: We can remove `Inc.` and just say `PlanetScale`.

File: content/en/docs/resources/presentations.md [hunk 2]
```diff
@@ -4,6 +4,34 @@ weight: 1
...
+
+{{< youtube id="upmByX8uD6E" autoplay="false" >}}
+
+Vitess: Introduction and New Features - Deepthi Sigireddi & Matt Lord & Rohit Nayak, PlanetScale, Inc.
```
Comment (deepthi) [inline content/en/docs/resources/presentations.md]:
```suggestion
Vitess: Introduction and New Features - Deepthi Sigireddi, Matt Lord & Rohit Nayak, PlanetScale, Inc.
```

File: content/en/docs/resources/presentations.md [hunk 3]
```diff
@@ -4,6 +4,34 @@ weight: 1
...
+
+## Percona Live 2022
+
+Vitess VReplication: Standing on the Shoulders of a MySQL Giant - Matt Lord, PlanetScale, Inc.
+
+{{< youtube id="ltoJ8gxs0YU" autoplay="false" >}}
+
+## KubeCon China 2021
+
+Continuous Performance Benchmarking for Vitess - Alkin Tezuysal & Florent Poinsard & Manan Gupta, PlanetScale, Inc.
```
Comment (deepthi) [inline content/en/docs/resources/presentations.md]:
```suggestion
Continuous Performance Benchmarking for Vitess - Alkin Tezuysal, Florent Poinsard & Manan Gupta, PlanetScale, Inc.
```

File: content/en/docs/resources/presentations.md (hunk 2 above)
Comment (deepthi) [reply content/en/docs/resources/presentations.md]: This is a nit, either way is fine.

File: content/en/docs/contributing/code-reviews.md [hunk 4]
```diff
@@ -91,6 +92,12 @@ During discussions, you can also refer to somebody using the *@username* syntax
 
 If you want to receive notifications even when you aren't mentioned, you can go to the [repository page](https://github.com/vitessio/vitess) and click *Watch*.
 
+## Reviewing a Pull Request
+
+The [Vitess bot](https://github.com/apps/vitess-bot) will comment a check-list on every pull request which contains a list of things to verify as a reviewer.
```
Comment (deepthi) [inline content/en/docs/contributing/code-reviews.md]:
```suggestion
The [Vitess bot](https://github.com/apps/vitess-bot) will add a comment with a review check-list on every pull request.
```

File: content/en/docs/contributing/code-reviews.md [hunk 5]
```diff
@@ -91,6 +92,12 @@ During discussions, you can also refer to somebody using the *@username* syntax
...
+The reviews should go through this list and mark the items as checked as they go along. If anything is incomplete, changes to the PR can be requested until
```
Comment (deepthi) [inline content/en/docs/contributing/code-reviews.md]:
```suggestion
Reviewers should go through this list and mark the items as checked as they go along. If anything is incomplete, changes to the PR can be requested until 
```

File: content/en/docs/contributing/code-reviews.md [hunk 6]
```diff
@@ -91,6 +92,12 @@ During discussions, you can also refer to somebody using the *@username* syntax
 
 If you want to receive notifications even when you aren't mentioned, you can go to the [repository page](https://github.com/vitessio/vitess) and click *Watch*.
 
+## Reviewing a Pull Request
+
+The [Vitess bot](https://github.com/apps/vitess-bot) will comment a check-list on every pull request which contains a list of things to verify as a reviewer.
+The reviews should go through this list and mark the items as checked as they go along. If anything is incomplete, changes to the PR can be requested until 
+all the points on the checklist are satisfied.
```
Comment (deepthi) [inline content/en/docs/contributing/code-reviews.md]:
```suggestion
all the items on the checklist are satisfied.
```

Comment (deepthi) [reply content/en/docs/contributing/code-reviews.md]: You have `check-list` once and `checklist` here. For the sake of consistency, you should choose one of the two and use the same in both places.

**Review by deepthi — APPROVED** [review]
Thank you for doing this!
You may merge once requested changes are complete.

File: content/en/docs/resources/presentations.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/resources/presentations.md]: Fixed.

File: content/en/docs/resources/presentations.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/docs/resources/presentations.md]: Fixed.

File: content/en/docs/resources/presentations.md (hunk 3 above)
Comment (GuptaManan100) [reply content/en/docs/resources/presentations.md]: Fixed

File: content/en/docs/contributing/code-reviews.md (hunk 4 above)
Comment (GuptaManan100) [reply content/en/docs/contributing/code-reviews.md]: Fixed

File: content/en/docs/contributing/code-reviews.md (hunk 5 above)
Comment (GuptaManan100) [reply content/en/docs/contributing/code-reviews.md]: Fixed

File: content/en/docs/contributing/code-reviews.md (hunk 6 above)
Comment (GuptaManan100) [reply content/en/docs/contributing/code-reviews.md]: Fixed. I went with `checklist`.

**Comment by rsajwani** [comment]
I see some of the youtube frames on presentation page are small .. is there a way to make them consistent. e.g
https://vitess.io/docs/resources/presentations/#percona-live-europe-2017

**Review by rsajwani — COMMENTED** [review]
LGTM

**Pull request merged**